deploy/sre-prometheus/100-control-plane-resizing.PrometheusRule.yaml

```diff
@@ -90,33 +90,36 @@ spec:
       ## Calculates CPU usage per master node as 100 – idle%.
       ## Converts each sample to 1 if usage >80%, 0 otherwise.
       ## avg_over_time(...[24h:5m]) computes the fraction of the last 24h each node was above 80% CPU.
-      - expr: |
-          avg_over_time(
-            (
-              (
-                (1 - avg by (instance) (
-                  rate(node_cpu_seconds_total{mode="idle"}[5m])
-                )) * 100
-                AND on(instance) label_replace(kube_node_role{role="master"}, "instance", "$1", "node", "(.+)")
-              ) > bool 80
-            )[24h:5m]
-          )
+      - expr: avg_over_time (
+                (
+                  (
+                    100 * sum by (instance) (
+                      rate( node_cpu_seconds_total{mode!="idle"}[5m] )
+                    )
+                    /
+                    sum by (instance) (
+                      rate( node_cpu_seconds_total[5m] )
+                    )
+                    AND on(instance) label_replace(
+                      kube_node_role{role="master"},"instance", "$1", "node", "(.+)"
+                    )
+                  ) > bool 80
+                )[24h:5m]
+              )
         record: sre:node_control_plane:cpu_usage_ebb
       ## Calculates memory usage per master node as (1 – free/total) × 100.
       ## Converts each sample to 1 if usage >80%, 0 otherwise.
       ## avg_over_time(...[24h:5m]) computes the fraction of the last 24h each node was above 80% memory.
-      - expr: |
-          avg_over_time(
-            (
-              (
-                1 - (
-                  (node_memory_MemFree_bytes + node_memory_Buffers_bytes + node_memory_Cached_bytes)
-                  / node_memory_MemTotal_bytes
-                )
-              ) * 100 > bool 80
-              AND on(instance) label_replace(kube_node_role{role="master"}, "instance", "$1", "node", "(.+)")
-            )[24h:5m]
-          )
+      - expr: avg_over_time(
+                (
+                  (
+                    node_memory_Active_bytes/node_memory_MemTotal_bytes*100
```

Comment:
The `node_memory_Active_bytes` represents the active memory from the kernel. It is not exact the used memory on node.
We still need to use the `total - ( free + buffer + cache )` here